portafly/src/i18n/locales/en/shared.yml

```diff
@@ -29,4 +29,14 @@ format:
   lowercase: "{{text, lowercase}}"
   lowercase_description: Formats a text as lowercase
   format_description: Utilities to format text
-
+shared_elements:
+  bulk_selector: {{number}} selected
+  bulk_selector_description: Label for the bulk selector to indicate the number of selected items. This label is visible only when items are selected.
+  send_button: Send
+  send_button_description: Label to perform the sending of items
+  delete_button: Delete
+  delete_button_description: Label to perform the deletion of selected items
+  delete_button_tooltip: Select at least an item to enable the option to delete.
```

Comment:
@porueesq "delete option" instead of "the option to delete" looks like a more common UI string, what do you think?

Reply:
Yes, it's simpler. I'll modify it.